fs/spiffs/src/spiffs_vfs.c

```diff
@@ -1187,7 +1187,10 @@ static int spiffs_fstat(FAR const struct file *filep, FAR struct stat *buf)
     }
 
   spiffs_unlock_volume(fs);
-  return spiffs_map_errno(ret);
```

Comment:
What is the `ret` and returned value here? I was thinking maybe it's worth updating `spiffs_map_errno` instead of the current change (just an idea to consider)?

Reply:
It's defined [here](https://github.com/tmedicci/incubator-nuttx/blob/546dd1c4c4f2eb376ccc639dbf0b4d33346231e1/fs/spiffs/src/spiffs_vfs.c#L178) and it just discards SPIFFS internal errors (but it doesn't "normalize" the return values). Once it's used in other places in the code, I didn't want to mess with it.